### DISCUSSION TITLE
ISCE

### DISCUSSION THREAD

**Discussion by nadiahassanpour** [isce-framework/isce2#567] [Q&A]
Completed Parsing the Configuration file
Functions to be executed:
['Function-1', 'Function-2']
Dear members, I got this error when I performed  ./run_all in Linux Terminal, my SLCs and my orbit files are completely correct, but I don't know what is problem. 
I would be happy if someone help me.
thanks

**Comment by SMJMirza** [comment]
Hi Nadia,

I cannot understand clearly what your problem is in your message. Could you please explain it in more detail? Or put a screenshot of error in the terminal here?

Sincerely,
Sayyed
